src/main/java/core/basesyntax/service/FileService.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface FileService {
+    List<String> read(String fileName);
```

Comment:
The `read` method should be named more descriptively to reflect what it reads. For example, `readFromFile` would be clearer.